data/json/items/armor/cloaks.json

```diff
@@ -26,6 +26,132 @@
       }
     ]
   },
+  {
+    "id": "pride_flag",
+    "repairs_like": "sheet",
+    "type": "ARMOR",
+    "name": { "str": "pride flag" },
+    "description": "A large rainbow flag, The colors reflect the diversity of the LGBT community and the spectrum of human sexuality and gender.",
+    "weight": "3404 g",
+    "volume": "2 L",
+    "price": 5000,
+    "price_postapoc": 50,
+    "to_hit": -1,
+    "material": [ "cotton" ],
+    "symbol": "]",
+    "looks_like": "sheet",
+    "color": "pink",
+    "warmth": 20,
+    "material_thickness": 0.3,
+    "environmental_protection": 1,
+    "flags": [ "OVERSIZE", "OUTER", "ALLOWS_NATURAL_ATTACKS" ],
+    "armor": [
+      {
+        "encumbrance": 50,
+        "coverage": 60,
+        "covers": [ "torso", "arm_l", "arm_r", "hand_l", "hand_r", "leg_l", "leg_r", "foot_l", "foot_r" ]
+      }
+    ],
+    "variant_type": "generic",
+    "variants": [
+      {
+        "id": "agender_pride_flag",
+        "name": { "str": "agender flag" },
+        "description": "A large green grey and black agender flag.  The black and white stripes represent complete absence of gender, grey represents being semi-genderless, and green represents non-binary gender.",
```

Comment:
More of a personal opinion here, but if you're adding inherently political stuff such as this, you may want to at least not elaborate the meaning of it as it's just bound to trigger some people. So for example, the description that avoids doing that would be "A horizontally striped flag with seven stripes: black, gray, white, green, white, gray, and black, in that order.".

Reply:
Some might think just having those flags in might trigger some people :)

The USA flag is clearly described as such, IIRC, if we clarify it we should also clarify those.

Reply:
It's flag, seems pretty normal to describe what the flag means.

Reply:
> Some might think just having those flags in might trigger some people :)

True, but as long as the text doesn't try to to rub any ideas into one's face (irregardless of how popular/unpopular the idea in question may be), I think it's just overreacting.

> The USA flag is clearly described as such, IIRC, if we clarify it we should also clarify those.

Well, in comparison the description just simply states that it is an American flag, rather than going as far as explaining what exactly each part of it means. So yeah, just stating that it is an "agender flag" should be enough.

Reply:
Feel free to add further description to the american flag if you feel it adds any interest. I don't think describing an object in the real world as it exists in the real world is objectionable.

Reply:
> Feel free to add further description to the american flag if you feel it adds any interest. I don't think describing an object in the real world as it exists in the real world is objectionable.

Seems like the implication of this is that whether a certain wording has the potential to offend someone or not is irrelevant as long as it is necessary to accurately describe what an item is. Am I right on this?

Reply:
Unless the descriptions here are factually wrong, or there are grammar or spelling mistakes they have a fine level of details. They're basically a list of gender, I really can't imagine how anyone could see something wrong with that.

Reply:
That's pretty disingenuous, but if you'd like to hear it more directly, we really don't care about people who are so easily offended that they can't bear to even be reminded that non-binary genders exist, without any obligation to interact with the concept in any way.

Reply:
>They're basically a list of gender, I really can't imagine how anyone could see something wrong with that.

It is not an universal belief or a solidly proven fact that all of the things described actually exist besides the fact that there's some people claiming to be that, so to anyone who does not, it'd be a bunch of nonsense.

Reply:
And yet regardless of if you believe that a purely conceptual category exists (what?), that remains what the flag represents, and the description makes no attempt to force you to interact with this data, so feel free to shed a soft tear and close the examine window.

Reply:
> And yet regardless of if you believe that a purely conceptual category exists (what?), that remains what the flag represents, and the description makes no attempt to force you to interact with this data, so feel free to shed a soft tear and close the examine window.

Reading already counts as an interaction, though. One can't just read something and immediately forget that they did if they didn't like it, so reading it ends up in the person who read it either agreeing with the fact that mentioned stuff exists, or getting a not-quite-comfortable "wait, what the hell did I just read?" or similar thought, because it implies the existence of the things that they just do not believe in in the first place, in the form of plain facts. I'm supportive of getting people to clear up their own misinformation if it's actually that (such as people believing they could make Molotov cocktails from common booze), but in this specific case you can't say for certain which one is true; simply said, neither one is proven.

Merely knowing that there are people who call themselves that doesn't count; as an example, some people may claim to be the Napoleon Bonaparte and people generally aren't surprised by the existence of such people, yet it's obviously a false claim, so if a person would read anything implying that the above fact may be true (such as "I was born on 15 August 1769; I'm an Emperor of the French and have recently escaped from Saint Helena island" that they know was written by someone who is living today), the person reading it would get cognitive dissonance. Obviously the player or the player character may or may not have similar delusional opinions, but I think it'd be wrong to assume that they do and have it result in an outright insane look on some things.

Reply:
Again, I truly don't care if someone's feelings are hurt because the game neutrally acknowledges how some people categorize themselves. I don't know how to make it clearer. If those people are alienated from the game, I consider that a very good outcome and hope they never again get any enjoyment out of something I worked to create.

Reply:
> Again, I truly don't care if someone's feelings are hurt because the game neutrally acknowledges how some people freely categorize themselves.

Why do you not, though? Especially when not doing so would cause discomfort to no one at all.

>If those people are alienated from the game, I consider that a very good outcome and hope they never again get any enjoyment out of something I worked to create.

That is just counterproductive. Games are made for as many people as possible to enjoy, not the other way around. I get that tastes may vary, but artificially placing limits on who could and could not enjoy doesn't seem right.

Reply:
Please you don't actually have a point, so stop here. This PR adds flags that exist irl and describe what they are, that's it and there's really no reason to be opposed to that.

Reply:
> Please you don't actually have a point, so stop here. This PR adds flags that exist irl and describe what they are, that's it and there's really no reason to be opposed to that.

I'm absolutely sure that I do have valid points here, to the most of which none of the people who have posted here have even bothered to respond. In turn, *I* have made my response to pretty much everything, though honestly I didn't even get much in the way of that. Of course you *can* ignore me or any other people for that matter, but the point still stands that you're pushing through certain political ideas only while pretending that you aren't, and some people may not like that. The essence of the responses I got (at least from I-Am-Erk) is pretty much "I personally don't like people who don't agree with this, so I don't care how they'll feel about it" while you keep saying over and over again that those are objectively true descriptions. Are any of those valid points? Well, you get to decide now, I just don't care at that point.

I'm not gonna post more as I'm not someone who would want to inconvenience people who don't agree with them on something as much as possible just for the sake of it (like I-Am-Erk claims to do). You disagreed, well, it's fine, I'm not gonna bug you any more on this, but bear in mind that you're in fact just getting me to shut up rather than actually proving me wrong. That's simply not a constructive method of discussing things.

Reply:
I take immediate, and complete, issue with the idea that we need to carefully vet language that simply describes the state of the world for fear of alienating people who refuse to accept, not even the lived reality of actual people, but *even the mention of them claiming that lived reality*. People who are that opposed to another person's right to free expression do not deserve any credence or attention whatsoever, and frankly I am incredibly disgusted that you are so intent on defending a person's right to not even see a vague reference to someone else's beliefs that disagree with theirs. This is identical to complaining that there is a mention of a person calling themselves a furry in game, because you believe furries are actually just into bestiality. Correct or not, nobody cares. Let people go do their harmless things and, if it's that hard being reminded that someone out there is doing something harmless that you don't accept, get fucked.

I'm done, but I'll just say you've gone from "somewhat respected, I'll listen to this person" to "this person appears to be a complete asshole" in my esteem. Not worth a ton, but you did work hard to achieve it, so you deserve to know you managed it.